pkg/cloudprovider/aws/fake/ec2api.go

```diff
@@ -159,7 +159,7 @@ func (e *EC2API) DescribeInstanceTypesPagesWithContext(ctx context.Context, inpu
 					DefaultVCpus: aws.Int64(2),
 				},
 				MemoryInfo: &ec2.MemoryInfo{
-					SizeInMiB: aws.Int64(8),
+					SizeInMiB: aws.Int64(8 * 1024),
```

Comment:
Could you do `SizeInGB` here?

Reply:
Unfortunately this is what EC2 returns, that's all you get. And I assume you mean SizeInGib not SizeInGb :)